SOLID/S/DY/GoodSolution/Interfaces/StorableInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Solid\S\DY\GoodSolution\Interfaces;
+
+interface StorableInterface
```

Comment:
Just add few comments regarding usage of this interface. Empty interface says nothing )

Reply:
from it implementation could guess that it will contain getName()